--- fluidsynth-1.1.8/configure.ac.orig	2017-10-12 17:32:26.000000000 +0200
+++ fluidsynth-1.1.8/configure.ac	2017-12-17 12:31:42.766731300 +0100
@@ -28,7 +28,7 @@
 dnl if any interfaces have been added: lt_age++
 dnl if any interfaces have been removed/changed (compatibility broken): lt_age=0
 
-m4_define([lt_current], [6])
+m4_define([lt_current], [7])
 m4_define([lt_revision], [1])
 m4_define([lt_age], [6])
 
