--- fluidsynth-1.1.5/src/Makefile.am.orig	2011-09-04 09:38:58.000000000 +0200
+++ fluidsynth-1.1.5/src/Makefile.am	2011-12-18 16:12:19.962675709 +0100
@@ -40,6 +40,9 @@ if OSS_SUPPORT
 fluid_oss = drivers/fluid_oss.c
 endif
 
+if MIDISHARE_SUPPORT
+fluid_midishare = drivers/fluid_midishare.c
+endif
 
 # if LASH_SUPPORT || LADCCA_SUPPORT (Makefile supports OR?)
 if LASH_SUPPORT
@@ -90,6 +93,7 @@ libfluidsynth_la_SOURCES = \
 	$(fluid_jack) \
 	$(fluid_ladspa) \
 	$(fluid_lash) \
+	$(fluid_midishare) \
 	$(fluid_oss) \
 	$(fluid_portaudio) \
 	$(fluid_pulse) \
--- fluidsynth-1.1.5/acinclude.m4.orig	2011-09-04 09:38:58.000000000 +0200
+++ fluidsynth-1.1.5/acinclude.m4	2011-12-18 16:10:42.556005782 +0100
@@ -32,6 +32,8 @@ AC_DEFUN([AC_MIDISHARE],
          AC_MSG_WARN([ *** Could not find MidiShare.h, disabling MidiShare driver])
      fi	dnl  midishare.h header test
    fi	dnl  enable_midishare != no?
+
+   AM_CONDITIONAL([MIDISHARE_SUPPORT], [test $MIDISHARE_SUPPORT = 1])
 ])
 
 AC_DEFUN([AC_OSS_AUDIO],
